ibdgc.datacommons.io/portal/gitops.json

```diff
@@ -96,10 +96,10 @@
           "name": "Query"
         },
         {
-          "name": "Notebook Browser",
+          "icon": "query",
           "link": "/resource-browser",
           "color": "#a2a2a2",
-          "icon": "query"
+          "name": "Notebook Browser"
```

Comment:
do we also need navigation link in the gitops.json file? 
something like this https://github.com/uc-cdis/gitops-qa/blob/master/qa-ibd.planx-pla.net/portal/gitops.json#L109-L114

Reply:
The Navigation link is not configured on purpose to meet with the requirements mentioned in the ticket here [PXP-8632](https://ctds-planx.atlassian.net/browse/PXP-8632)